### PR TITLE
Fix typo pipline -> pipeline

### PR DESCRIPTION
**Pull request by hklai**

### CODE REVIEW

File: release/pipeline/configs/environment_config.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
     MFEST_FILE='build.xml',
     MFEST_COMMIT='master@{{{timestamp}}}',
     VERSION='{major}.{minor}.{patch}-pre{date}-{rc}',
-    GCS_BUILD_BUCKET='istio-release-pipline-data',
+    GCS_BUILD_BUCKET='istio-release-pipeline-data',
```
Comment (mattdelco) [inline release/pipeline/configs/environment_config.py]: Be sure to create the bucket.

Comment (mattdelco) [reply release/pipeline/configs/environment_config.py]: Line 46 also has an instance of "pipline", but it's too far away from any other change for github to allow me to comment on it.

Comment (hklai) [reply release/pipeline/configs/environment_config.py]: Yes I have already created the bucket.

Also updated line 46. Failed to take care of the capital P when I grepped.

Comment (mattdelco) [reply release/pipeline/configs/environment_config.py]: Hrm, the bucket didn't exist when I made the comment.  Anyway, it does seem to exist now.  I'm not sure if the location matters, but for now I'm doing to assume that multi-regional is a strict superset of a specific region (which is what the original used).

Comment (sebastienvas) [reply release/pipeline/configs/environment_config.py]: in which project was the bucket created ?

Comment (mattdelco) [reply release/pipeline/configs/environment_config.py]: istio-release.  This is also the project used for running the daily builds

**Comment by mattdelco** [comment]
/lgtm
though you can also kill a second bird with this stone by fixing the 4 instances of "quilification" in pipline/dags/istio_common_dag.py.

**Pull request merged**